Appraisals

```diff
@@ -101,5 +119,8 @@ appraise "active_support_7-0_redis_cache_store_pooled" do
 end
 
 appraise "redis_store" do
+  # Direct version requirement on connection_pool
+  # can be removed once https://github.com/rails/rails#56291 is fixed and released
+  gem "connection_pool", "~> 2.5"
```

Comment:
`redis_4`, `redis_5`, and `redis_store` gemfiles need it but it isn't because of redis, it's because of [activesupport being installed as a dev dependency](https://github.com/rack/rack-attack/blob/main/rack-attack.gemspec#L48) which seems unfortunate, but when trying to remove it lot of tests failed so I think we can work on that separately if we ever want to cc @grzuy

Reply:
Gotcha.
And we can't solve it by listing connection_pool 2.5 as an additional dev dependency in the gemspec, because that would affect tests that try testing when connection_pool is not present, right?